src/model/base.go

```diff
@@ -120,13 +120,15 @@ type Event struct {
 
 // Block - transformation block.
 type Block struct {
+	BlockKey
```

Comment:
Aby bol kod jednoduchsi, tak si v `Block` struct ukladame aj `ConfigId, BranchId .... ` aby sme to mali vzdy k dispozicii.